HOWTO-make-a-release.md

```diff
@@ -208,8 +223,9 @@ match in the .md5, .sha1, .sha256, and review the announcment file.
 *Do* send the auto-generated commits to the reviewer and await their
 approval.
 
-*Do not push* changes to the main source repo at this stage.
-(the main source repo being `git@github.openssl.org:openssl/openssl.git`)
+*Do not push* changes to the source repo at this stage.
+(the source repo being on of `git@github.openssl.org:openssl/openssl.git` or
+`git@github.openssl.org:openssl/premium.git`)
```

Comment:
@mattcaswell, your comment further down (near `git format-patch`) made me wonder if this is true any more, or needs an update as well...

Reply:
This is still correct. Pushing the changes to the source repo makes them public (at least for the public repo) and we do that quite late in the process.